pages/common/hydra.md

```diff
@@ -0,0 +1,36 @@
+# hydra
+
+> Online password guessing tool.
+> More information: <https://github.com/vanhauser-thc/thc-hydra>.
+
+- Start Hydra's wizard:
+
+`hydra-wizard`
+
+- Guess SSH credentials using a given username and a list of passwords:
+
+`hydra -l {{username}} -P {{path/to/wordlist.txt}} {{host_ip}} {{ssh}}`
```

Comment:
Are there any long forms of these options?

Reply:
No, all the options are in short form.